<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{
border: 1px solid #5c6266;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>998</width>
      <height>72</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame{ 
background: qlineargradient(spread:pad, x1:0, y1:0.96, x2:1, y2:1, stop:0 rgba(38, 184, 214, 255), stop:0.323454 rgba(137, 206, 221, 255), stop:0.623711 rgba(208, 247, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-bottom: 2px solid #5c6266;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	color: white;
    font-size: 24px;
    line-height: 60px;
    margin-left: 20px;
    background: transparent;
font-weight: bold;
</string>
     </property>
     <property name="text">
      <string>Медицинское приложение</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>101</width>
       <height>72</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="image.qrc">:/image/logo1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="min_btn">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>10</y>
       <width>46</width>
       <height>46</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#min_btn{
 	background-image: url(&quot;:/image/drug.png&quot;);
    background-repeat: no-repeat;
    background-position: center center;
}

#min_btn:hover {
    background-image: url(&quot;:/image/drug_act.png&quot;);
    background-repeat: no-repeat;
	background-position: center center;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="close_btn">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>10</y>
       <width>46</width>
       <height>46</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#close_btn{
 	background-image: url(&quot;:/image/pharmacy.png&quot;);
    background-repeat: no-repeat;
    background-position: center center;
}

#close_btn:hover {
    background-image: url(&quot;:/image/pharmacy_act.png&quot;);
    background-repeat: no-repeat;
	background-position: center center;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>74</y>
      <width>3</width>
      <height>531</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #5c6266;
min-width: 1px;
max-width: 1px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>73</y>
      <width>198</width>
      <height>522</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_3{
background: white;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>201</width>
       <height>531</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="menu3">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Пациенты</string>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="menu2">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Врачи</string>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="menu1">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Услуги</string>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="menu4">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Запись</string>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>195</width>
          <height>426</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>540</y>
      <width>591</width>
      <height>44</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="add">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Добавить </string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/add_hov.png</normaloff>:/image/add_hov.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del">
       <property name="styleSheet">
        <string notr="true">#add{
	background: qlineargradient(spread:pad, x1:0.053, y1:0.733, x2:1, y2:1, stop:0 rgba(0, 128, 	154, 255), stop:0.451724 rgba(82, 143, 156, 255), stop:1 rgba(172, 205, 212, 255));
	color:white;
  font-size: 19px;
   text-align: left;
   padding-left: 12px;
   border-radius: 21px;
   border-style: solid;
   border: 1px solid #5c6266;
   min-width:240px;
   min-height:40px;
   max-width:240px;
   max-height:40px;
}

#add:hover {
	background: qlineargradient(spread:pad, x1:0.053, y1:0.733, x2:1, y2:1, stop:0 rgba(1, 157, 189, 255), stop:0.451724 rgba(109, 188, 205, 255), stop:1 rgba(202, 241, 249, 255));
}</string>
       </property>
       <property name="text">
        <string>Удалить </string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/delete.png</normaloff>:/image/delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="edit">
       <property name="styleSheet">
        <string notr="true">#add{
	background: qlineargradient(spread:pad, x1:0.053, y1:0.733, x2:1, y2:1, stop:0 rgba(0, 128, 	154, 255), stop:0.451724 rgba(82, 143, 156, 255), stop:1 rgba(172, 205, 212, 255));
	color:white;
  font-size: 19px;
   text-align: left;
   padding-left: 12px;
   border-radius: 21px;
   border-style: solid;
   border: 1px solid #5c6266;
   min-width:240px;
   min-height:40px;
   max-width:240px;
   max-height:40px;
}

#add:hover {
	background: qlineargradient(spread:pad, x1:0.053, y1:0.733, x2:1, y2:1, stop:0 rgba(1, 157, 189, 255), stop:0.451724 rgba(109, 188, 205, 255), stop:1 rgba(202, 241, 249, 255));
}</string>
       </property>
       <property name="text">
        <string>Изменить </string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/change.png</normaloff>:/image/change.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>721</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">#lineEdit{
	font-size: 15px;
	padding-left: 15px;
	min-height: 30px;
	max-height: 30px;
	border: 2px solid #019dbd;
	border-right: 2px solid transparent;
	border-top-left-radius: 16px;
    border-bottom-left-radius: 16px;
  	color: #5c6266;
}

#lineEdit:hover {
	border: 2px solid #00d4ff;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="search">
       <property name="styleSheet">
        <string notr="true">#search{
	font-size: 15px;
	padding-left: px;
	min-width: 60px;
	max-width: 60px;
	min-height: 34px;
	max-height: 34px;
     background: qlineargradient(spread:pad, x1:0.053, y1:0.733, x2:1, y2:1, stop:0 rgba(0, 128, 	154, 255), stop:0.451724 rgba(82, 143, 156, 255), stop:1 rgba(172, 205, 212, 255));
	border-top-right-radius: 16px;
    border-bottom-right-radius: 16px;
}

#search:hover {
    background: qlineargradient(spread:pad, x1:0.053, y1:0.733, x2:1, y2:1, stop:0 rgba(1, 157, 189, 255), stop:0.451724 rgba(109, 188, 205, 255), stop:1 rgba(202, 241, 249, 255));
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/search.png</normaloff>:/image/search.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>151</y>
      <width>721</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QHeaderView {
    color: #5c6266;
    font:  14pt;
	border-bottom: 0.8px solid gray;
	border-color: black;
	font-weight: normal;
	background-color: rgba(255, 255, 255, 0.1);
    border-radius:0px;
}

QHeaderView::section {
    color: #5c6266;
  	font:  14pt;
	font-weight: normal;
    border-radius: 6px;
    border-color: rgba(255, 255, 255, 0.1);  
}

QTableView {
	border:none;
    background-color: rgba(255, 255, 255, 0.0);
}

QTableView::item {
		text-align: center;
        padding-left: 5px;
        padding-right: 5px;
  		font:  12pt;
        border: none;
        background-color: rgba(255, 255, 255, 0.3);
}

QTableView::item:selected:!active {
		border: none;
        color: #5c6266;
 		background:   rgba(208, 247, 255, 255);
}

QTableView::item:selected {
		color: black;
		border: none;
        background:   rgba(208, 247, 255, 255);
}


 


</string>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>75</number>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
